unit/solvers/smt2_incremental/smt_to_smt2_string.cpp

```diff
@@ -1,9 +1,8 @@
 // Author: Diffblue Ltd.
 
-#include <util/mp_arith.h>
+#include <util/mp_arith.h> // IWYU pragma: keep
 
 #include <solvers/smt2_incremental/ast/smt_commands.h>
-#include <solvers/smt2_incremental/ast/smt_logics.h>
```

Comment:
The logics defined in this header are used as part of the "Test smt_set_logic_commandt to string conversion" test in this file. Removing the include still compiles as it depends on a transitive include via `smt_commends.h`.

Reply:
I have now added `// IWYU pragma: keep` instead.